### ISSUE TITLE
Feature request: keep chord accompaniment within set octave range, even in presence of transpose

### ISSUE THREAD

**Issue by chrisfargen** [paulrosen/abcjs#391]
The code below generates accompaniment with missing notes.
```
T: Open strings, guitar (EADGBe)
M: 4/4
L: 1/4
K: Cmaj transpose=-12
%
"C" E, A, D G | B e z2 | e B G D | A, E, z2 ||
```
This is particularly problematic for guitar music, which is typically played one octave below what is written. (Other instruments that transpose similarly include bass guitar, banjo, cello, and upright bass.)

**Comment by paulrosen** [comment]
Ok, I'll get to this soon.

**Comment by paulrosen** [comment]
Will be fixed in version 6.0.0-beta.10

**Comment by chrisfargen** [comment]
Working great. Thanks Paul!
